.github/workflows/pr-1-ci.yml

```diff
@@ -77,24 +83,88 @@ jobs:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GH_REPO: ${{ github.repository }}
           BODY: |
-            :x: Automated testing cannot be run on this branch :x:
+            :x: Automated Reproducibility testing cannot be run on this branch :x:
             Source and Target branches must be of the form `dev-<config>` and `release-<config>` respectively, and `<config>` must match between them.
             Rename the Source branch or check the Target branch, and try again.
         run: gh pr comment --body '${{ env.BODY }}'
 
+  qa-ci:
+    # Run quick, non-HPC tests on the runner.
+    name: QA CI Checks
+    needs:
+      - commit-check
+    if: needs.commit-check.outputs.authorship != vars.GH_ACTIONS_BOT_GIT_USER_NAME
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write
+    steps:
+      - name: Checkout PR ${{ github.event.action.pull_request.number }}
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+          path: pr
+
+      - name: Checkout Tests
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          path: pytest
+
+      - name: Setup Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+          cache: pip
+
+      - name: Install requirements.txt
+        working-directory: ./pytest
+        run: pip install -r ./test/requirements.txt
+
+      - name: Get Final List of Markers
+        id: pytest-markers
+        run: |
```

Comment:
Just wanted to check we don't want to check the validity of the markers input to the CI.

I don't think we have the ability to know a priori what markers might be valid. And if we did limit them to a known set it would also limit the felxibility of the CI workflow in the future. 

I wanted to make sure we'd had this conversation and made this choice deliberately.

Reply:
Yeah I don't think we do want to check this. It'd be a bit of a pain (would probably have to checkout the `pytests` and get the markers through `grep`, or something). Also, `pytest` will note errors in the invocation as well, so it's not like the reason for the failure during a workflow will be entirely opaque.